.jenkins/pipelines/code-coverage.Jenkinsfile

```diff
@@ -5,23 +5,23 @@
 
 pipeline {
     agent {
-        label 'ACC-1804-DC2'
+        label 'Jenkins-Shared-DC2'
     }
     options {
         timeout(time: 300, unit: 'MINUTES')
     }
     parameters {
         string(name: "REPOSITORY", defaultValue: "deislabs")
         string(name: "BRANCH", defaultValue: "main", description: "Branch to build")
-        choice(name: "REGION", choices:['useast', 'canadacentral'], description: "Azure region for SQL test")
+        choice(name: "REGION", choices:['useast', 'canadacentral'], description: "Azure region for solutions test")
```

Comment:
Nit: I believe the region only pertains to the SQL solution test only, rather than "all solution tests"